### PR TITLE
Style/1.1 global

### PR DESCRIPTION
**Pull request by HugoWith**
## Objective
What:
👉 Styling fix 
👉 Micro interactions 
👉 Better responsiveness
👉 Rmv some unused code
Why:
👉 Better UI/UX

## Type

- [ ] Bug Fix
- [ ] New Feature
- [ ] Performance Tweaks
- [x] Style Tweaks
- [ ] Code Refactoring
- [ ] Documentation
- [ ] Tests

## Tested

✅💅

## Documented

- [ ] Have you documented in changed file using comments
- [ ] Have you added instructions to the README if needed?

### CODE REVIEW

File: src/components/header/components/Navigation.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // Render the navigation menu in the header
 
+import { useState } from 'react';
```
Comment (VirAlex) [inline src/components/header/components/Navigation.jsx:3]: Maybe remove this unused import 👍

File: src/components/header/components/Navigation.jsx [hunk 2]
```diff
@@ -201,9 +204,9 @@ const Navigation = ({ isMenuOpen, setIsMenuOpen, mobile, tablet }) => {
           {!mobile && <CartPicto />}
           {mobile && <p>Cart</p>}
           {/* Picto notification up the cart icon */}
-          {showCart?.length > 0 && (
+          {showCart?.length !== 0 && (
```
Comment (VirAlex) [inline src/components/header/components/Navigation.jsx:207]: Just to know why change it it was stuck?

File: src/components/searchresultpage/SCSS/searchResultsPage.scss [hunk 3]
```diff
@@ -237,14 +237,16 @@
...
 
-    li {
-      width: fit-content;
-    }
-  }
 }
+
+// @media only screen and (max-width: 480px) {
+//   .uic-HorizontalSlider-list {
+
```
Comment (VirAlex) [inline src/components/searchresultpage/SCSS/searchResultsPage.scss:246]: Maybe remove commented code 👍

**Review by VirAlex — CHANGES_REQUESTED** [review]

File: src/components/carousels/HitsCarousel.jsx [hunk 4]
```diff
@@ -49,12 +50,12 @@ const HitsCarousel = ({ hit, index }) => {
   return (
     <div
       className="item"
-      onMouseEnter={() => {
-        setHovered(true);
-      }}
-      onMouseLeave={() => {
-        setHovered(false);
-      }}
+      // onMouseEnter={() => {
```
Comment (matthew1809) [inline src/components/carousels/HitsCarousel.jsx:53]: Remove don't comment?

File: src/components/header/components/Navigation.jsx (hunk 2 above)
Comment (HugoWith) [reply src/components/header/components/Navigation.jsx:207]: Don't know TBH, I could see cart with 0 items in it. 
But I think we could do : 
`showCart?.length && ()`

**Comment by HugoWith** [comment]
@matthew1809 @VirAlex Thanks a lot for your comments ! Changes are done 😊

**Review by VirAlex — APPROVED** [review]
LGTM GG 👏

File: src/components/header/components/Navigation.jsx (hunk 2 above)
Comment (VirAlex) [reply src/components/header/components/Navigation.jsx:205]: Nice work on this side

**Pull request merged**